pac4j-cas/src/test/java/org/pac4j/cas/client/CasClientTests.java

```diff
@@ -127,23 +125,23 @@ public void testBackLogout() {
         casClient.setCallbackUrl(CALLBACK_URL);
         casClient.init(null);
         final MockWebContext context = MockWebContext.create().addRequestParameter(CasConfiguration.LOGOUT_REQUEST_PARAMETER, LOGOUT_MESSAGE)
-            .setRequestMethod(HTTP_METHOD.POST);
+            .setRequestMethod("POST");
```

Comment:
Yes, I could use the enum here and there to avoid using simple strings. I will take a look at that while adding the `PATCH` method.